igvc_platform/launch/lidar.launch

```diff
@@ -1,13 +1,18 @@
 <!-- lidar.launch -->
 <launch>
     <!-- Lidar Node -->
-    <node name="sweep_node"	pkg="sweep_ros"  type="sweep_node" output="screen">
-      <param name="serial_port"         type="string" value="/dev/igvc_lidar"/>
-      <param name="serial_baudrate"     type="int"    value="115200"/>
-      <param name="frame_id"            type="string" value="lidar"/>
-      <param name="rotation_speed"    type="int" value="3"/>
-      <param name="sample_rate"    type="int" value="1000"/>
-      <param name="distance_threshold"    type="double" value="50"/>
+    <node name="rplidarNode"              pkg="rplidar_ros"  type="rplidarNode" output="screen">
+        <param name="serial_port"         type="string" value="/dev/igvc_lidar"/>
+        <param name="serial_baudrate"     type="int"    value="256000"/><!--A3 -->
+        <param name="frame_id"            type="string" value="/lidar"/>
+        <param name="inverted"            type="bool"   value="false"/>
+        <param name="angle_compensate"    type="bool"   value="true"/>
+        <param name="scan_mode"           type="string" value="Boost"/>
```

Comment:
If this can/should change it would be nice to know the options and what they mean.

Reply:
Good catch, I forgot to add those in